packages/flutter/test/material/bottom_app_bar_test.dart

```diff
@@ -85,13 +85,17 @@ void main() {
     );
   }, skip: isBrowser); // https://github.com/flutter/flutter/issues/44572
 
-  testWidgets('color defaults to Theme.bottomAppBarColor', (WidgetTester tester) async {
+  testWidgets('color defaults to BottomAppBarTheme.color', (WidgetTester tester) async {
```

Comment:
Changing something in /test makes this a breaking change, which is harder to land, so you should try to make the change non-breaking (see comment above)

Reply:
Done